frontend/src/common/PageWithTabs.tsx

```diff
@@ -19,30 +19,35 @@ export interface PageTab {
 
 export default function PageWithTabs({
   title,
+  subheading,
   tabs,
   actionButtonTitle = '',
   displayActionButton = false,
   actionButtonOnClick,
   requiredUrlParams = {},
   showCopyButton = false,
-  textToCopy = '',
+  titleToCopy = '',
+  subheadingToCopy = '',
   sourceModelId = '',
 }: {
   title: string
+  subheading?: string
```

Comment:
I think this as an optional type looks like a typo?

Reply:
Not a typo - we don't always need a subheading eg the schemas page.